composer.json

```diff
@@ -197,6 +197,7 @@
             "newsletter-bundle/src/Resources/contao/"
         ],
         "files": [
+            "core-bundle/src/Resources/functions/symfony-polyfill.php",
```

Comment:
I'd love to find a better place for this… but not sure what would be appropriate. Is there an example somewhere else?

Reply:
Take a look at the Symfony polyfill packages: https://github.com/symfony/polyfill-php80 I can create a separate package for our polyfills, too.

Reply:
I don’t think there is. Symfony polyfills use `classmap` but this is only possible to polyfill native classes because non-native classes would conflict with the autoload settings of the actual classes.

Reply:
See https://github.com/contao/polyfill-symfony. A separate package has the advantage that we can remove it again in Contao 4.12 and third-party bundles can still require it if they need to.

Reply:
Changed in 914c85b758b4001e9a66943b4872c42938a5511b.